### ISSUE TITLE
PACT jvm provider maven POM equivalent in Junit

### ISSUE THREAD

**Issue by garriual** [DiUS/pact-jvm#204]
This is a follow up from https://groups.google.com/forum/#!topic/pact-support/bDlnEBC2rDY.
I am unable to implement PACT verification from JUnit provider library, and wonder if you guys can provide some example. Currently our application uses Spring as the application framework, which uses dependecy injection most of the time in our integration test. If you guys can also give some example if using Spring dependency injection for the filtering part (setting the header from the authorization service that we have) that would be great. 

Thanks,
Gary

**Comment by uglyog** [comment]
I've implemented the JUnit request filters. You'll be able to do something like:

```java
    private String token;

    @TargetRequestFilter
    public void exampleRequestFilter(HttpRequest request) {
      if (token == null) {
        token = <code to fetch auth token>;
      }
      request.addHeader("Authorization", token);
    }
```

This should replace the maven plugins and property files you had to previously use.

**Comment by uglyog** [comment]
This has been released with version 3.2.3
